site/assets/css/fish_style.css

```diff
@@ -263,9 +263,7 @@ h3 {
     image-rendering: -o-crisp-edges;
     image-rendering: -webkit-optimize-contrast;
     image-rendering: -webkit-crisp-edges;
-    image-rendering: pixelated;
-    image-rendering: optimize-contrast; 
-    image-rendering: crisp-edges;
+    image-rendering: optimize-contrast;
```

Comment:
I'm use [WebStorm](https://www.jetbrains.com/webstorm/) and mark it as unnecessary code. Maybe WS lied?